File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        String even = "";
        String odd = "";
        for(int i = 0; i < num_list.length; i++) {
            if(num_list[i]%2==0) {
                even += Integer.toString(num_list[i]);
            } else {
                odd += Integer.toString(num_list[i]);
            }
        }
        return Integer.parseInt(even) + Integer.parseInt(odd);
    }
}

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    let even = [], odd = [];
    for(let i = 0; i < num_list.length; i++) {
        if(num_list[i]%2==0)
            even.push(num_list[i].toString());
        else
            odd.push(num_list[i].toString());
    }
    return parseInt(even.join('')) + parseInt(odd.join(''));
}